tools/tpm2_create.c

```diff
@@ -164,10 +166,10 @@ int setup_alg()
     return 0;
 }
 
-int create(TSS2_SYS_CONTEXT *sapi_context)
```

Comment:
Probably I wouldn't mix cleanups like the create() return value in this commit, it makes harder to review. But I don't mind to have them either.

Reply:
Yep my patch granularity was terrible. I just got in the habit of blindly collapsing my commits.